logstash-core/src/main/java/org/logstash/plugins/factory/PluginsFactoryExt.java

```diff
@@ -0,0 +1,259 @@
+package org.logstash.plugins.factory;
+
+import co.elastic.logstash.api.*;
+import org.jruby.*;
+import org.jruby.anno.JRubyClass;
+import org.jruby.anno.JRubyMethod;
+import org.jruby.javasupport.JavaUtil;
+import org.jruby.runtime.ThreadContext;
+import org.jruby.runtime.builtin.IRubyObject;
+import org.logstash.RubyUtil;
+import org.logstash.config.ir.PipelineIR;
+import org.logstash.config.ir.compiler.*;
+import org.logstash.config.ir.graph.Vertex;
+import org.logstash.execution.ExecutionContextExt;
+import org.logstash.instrument.metrics.AbstractMetricExt;
+import org.logstash.instrument.metrics.AbstractNamespacedMetricExt;
+import org.logstash.instrument.metrics.MetricKeys;
+import org.logstash.plugins.PluginLookup;
+
+import java.util.*;
+
+@JRubyClass(name = "PluginFactory")
+public final class PluginsFactoryExt extends RubyBasicObject
```

Comment:
the convention seems to be adding an Ext suffix - thus `PluginFactoryExt` (wout the 's')

Reply:
Done